client/performance-profiler/components/core-web-vitals-display/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import { PerformanceMetrics } from 'calypso/performance-profiler/types/performance-metrics';
+import {
+	metricsNames,
+	metricsTresholds,
+	mapThresholdsToStatus,
+} from 'calypso/performance-profiler/utils/metrics';
+import { MetricScale } from '../metric-scale';
+import { MetricTabBar } from '../metric-tab-bar';
+import './style.scss';
+import { StatusIndicator } from '../status-indicator';
+
+export const CoreWebVitalsDisplay = ( props: PerformanceMetrics ) => {
+	const translate = useTranslate();
+	const [ activeTab, setActiveTab ] = useState< string >( 'lcp' );
+
+	const { name: metricName, displayName } = metricsNames[ activeTab as keyof typeof metricsNames ];
+	const value = props[ activeTab as keyof PerformanceMetrics ];
+	const valuation = mapThresholdsToStatus( activeTab as keyof typeof metricsTresholds, value );
+
+	const displayValuation = ( valuation: string ) => {
+		switch ( valuation ) {
+			case 'good':
+				return 'is good';
+			case 'needsImprovement':
+				return 'needs improvement';
+			case 'bad':
+				return 'is bad';
+			default:
+				return 'unknown';
+		}
+	};
+
+	const { good, needsImprovement } = metricsTresholds[ activeTab as keyof typeof metricsTresholds ];
+	const formatUnit = ( value: number ) => {
+		if ( [ 'lcp', 'fcp', 'ttfb' ].includes( activeTab ) ) {
+			return +( value / 1000 ).toFixed( 2 );
+		}
+
+		return value;
+	};
+
+	const displayUnit = () => {
+		if ( [ 'lcp', 'fcp', 'ttfb' ].includes( activeTab ) ) {
+			return 'seconds';
+		}
+
+		if ( [ 'inp' ].includes( activeTab ) ) {
+			return 'milliseconds';
+		}
+
+		return '';
+	};
+
+	return (
+		<div className="core-web-vitals-display">
+			<MetricTabBar activeTab={ activeTab } setActiveTab={ setActiveTab } { ...props } />
+			<div className="core-web-vitals-display__details">
+				<div className="core-web-vitals-display__description">
+					<span className="core-web-vitals-display__description-subheading">
+						{ translate( "Your site's %(displayName)s %(valuation)s", {
+							args: {
+								displayName: displayName.toLowerCase(),
+								valuation: displayValuation( valuation ),
+							},
+						} ) }
+					</span>
+					<MetricScale metricName={ activeTab } value={ value } valuation={ valuation } />
+					<div className="core-web-vitals-display__ranges">
+						<div className="range">
+							<StatusIndicator speed="good" />
+							<div className="range-description">
+								<div className="range-heading">{ translate( 'Fast ' ) }</div>
+								<div className="range-subheading">
+									{ translate( '0–%(to)s %(unit)s', {
+										args: { to: formatUnit( good ), unit: displayUnit() },
+									} ) }
```

Comment:
As this is a simpler phrase I think it might work without a dictionary.

But if you are creating one for the other case it might worth to just reuse the logic.

Reply:
In this case, the two strings that need to be translated are really just the `seconds` and `milliseconds`. I wrapped these in translate and left the templating as is.